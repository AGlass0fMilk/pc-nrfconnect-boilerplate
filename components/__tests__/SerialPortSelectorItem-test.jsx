import React from 'react';
import renderer from 'react-test-renderer';
import {
    decorateSerialPortSelectorItem,
} from '../SerialPortSelectorItem';

describe('SerialPortSelectorItem', () => {
    it('should implement component decorator', () => {
        expect(decorateSerialPortSelectorItem).toBeInstanceOf(Function);
    });
});

describe('decorateSerialPortSelectorItem', () => {
    it('should render component', () => {
        const SerialPortSelectorItem = decorateSerialPortSelectorItem('div');
        expect(renderer.create(
            <SerialPortSelectorItem />,
        )).toMatchSnapshot();
    });
});
