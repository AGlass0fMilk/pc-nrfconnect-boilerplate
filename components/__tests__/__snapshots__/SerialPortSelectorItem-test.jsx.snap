exports[`decorateSerialPortSelectorItem should render component 1`] = `<div />`;
